### PR TITLE
Improve man and html page rendering

### PR DESCRIPTION
**Pull request by MikeFultonDev**
Primarily formatting changes for the online help so it renders better. In particular, you need 2 spaces after a word and it's definition for it to be rendered properly.
Also fixed a place where *.yml may not exist in generate and it spits out a disconcerting warning.
Added back the common.sh include now that things seem to have settled.

### CODE REVIEW

**Review by AnthonyGiorgio — APPROVED** [review]

File: bin/zopen-generate [hunk 1]
```diff
@@ -119,7 +119,7 @@ touch ${buildenv} && chtag -tc 819 ${buildenv}
 
 cp "${MYDIR}/../data/CONTRIBUTING.md" "${name}port/CONTRIBUTING.md"
 mkdir -p "${name}port/.github/workflows"
-cp "${MYDIR}/../data/*.yml" "${name}port/.github/workflows/"
+cp "${MYDIR}/../data/*.yml" "${name}port/.github/workflows/" 2>/dev/null # no yml files yet?
```
Comment (IgorTodorovskiIBM) [inline bin/zopen-generate:122]: Oh, I must have forgotten to add the data files. Opened PR https://github.com/ZOSOpenTools/meta/pull/550

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Pull request merged**